src/components/app/Details.css

```diff
@@ -0,0 +1,6 @@
+.Details {
+  display: flex;
+  flex-direction: column;
+
```

Comment:
I like the empty line, because it makes it more obvious that the 2 parts of this rule are for 2 different things:
`flex: auto` defines the relationship with its parent container, while `display: flex` and `flex-direction: column` defines the relationship with its children.

That said I'd be tempted to switch the order: start with `flex: auto`, and then with the children's layout. What do you think?